CMakeLists.txt

```diff
@@ -332,7 +332,9 @@ if (NOT HAVE_CXX_ATOMICS_WITHOUT_LIB)
     endif()
     set(LDFLAGS "${LDFLAGS} -latomic")
   else()
-    message(FATAL_ERROR "Host compiler appears to require libatomic, but cannot find it.")
+    message(STATUS "Host compiler appears to require libatomic, but cannot find it.")
+    message(STATUS "Enabling mutex lock workaround.")
+    set(MEASUREMENT_MUTEX_WORKAROUND ON)
```

Comment:
Let's prefix this with `VAST_`, as we do with all other variables that wind up in source code.